Makefile

```diff
@@ -96,7 +96,10 @@ $(OUT)/storage_default.tz: $(shell find ligo -name '*.mligo')
 			; y_token_address = (\"$(y_token_address)\" : address) \
 	    })" --output-file $(OUT)/storage_default.tz
 
-prepare_lib: every
+$(OUT)/liquidity_mining.tz: ligo/liquidity_mining.mligo
+	$(BUILD) ligo/liquidity_mining.mligo main --output-file $@
+
+prepare_lib: every $(OUT)/liquidity_mining.tz
```

Comment:
If you take a look at `ci.nix` in the `build-ligo` what is used is `make every`.

Note that `build-ligo`'s definition is also used by haskell's steps to compile the source code, which is why CI is failing for you.

I'd suggest reverting `prepare_lib` and add `liquidity_mining.tz` to the `every` target instead, since the latter only purpose is indeed for tests.

Reply:
Thanks, makes sense indeed.